app/src/main/java/App.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+
+import javax.imageio.ImageIO;
+import javax.imageio.ImageReader;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.awt.image.RenderedImage;
+import java.io.File;
+import java.io.IOException;
+import java.lang.*;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class App {
+
+// ./gradlew run --args '/Users/matthewsimms/Documents/projects/401/bitmap-transformer/app/src/main/resources/BitMap.bmp /Users/matthewsimms/Documents/projects/401/bitmap-transformer/app/src/main/resources/NewBitMap.bmp flipImage'
+    public static void main(String[] args) throws IOException {
+        System.out.println(args[0]);
```

Comment:
Testing to see what args[] is? It's great to leave things like this in your code for me, helps me see how you approached things